enterprise/backend/src/metabase_enterprise/serialization/v2/load.clj

```diff
@@ -53,8 +53,8 @@
 
 (defn- path-error-data [error-type expanding path]
   (let [last-model (:model (last path))]
-    {:path       path
-     :deps-chain expanding
+    {:path       (mapv (partial into {}) path)
+     :deps-chain (set (map #(mapv (partial into {}) %) expanding))
```

Comment:
Doesn't order matter for the chain?

Reply:
IDK to be honest, but I can drop set there :)